scripts/increment-cargo-version.sh

```diff
@@ -108,6 +110,15 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   done
 done
 
+# Update all the documentation references
+for Doc_update in "${Doc_updates[@]}"; do
```

Comment:
nit: let's rename these variables to something a little more clear:
I suggest `Doc_updates` -> `markdownFiles`, and `Doc_update` -> `file`